src/constants.js

```diff
@@ -150,6 +148,6 @@ export const errorCodes = {
   FORBIDDEN: 403,
 };
 export const DAEMON_USER = {
-  "name": "daemon",
-  "uuid": "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB"
-}
+  name: "daemon",
+  uuid: "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB",
```

Comment:
is the user UUID for daemon user fixed? Ensure that it is created by OMRS and with a fixed UUID

Reply:
Seems like its hardcoded in OMRS - 	[protected static final String DAEMON_USER_UUID = "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB";](https://github.com/openmrs/openmrs-core/blob/6e43defc403de51799946497e76efa86207b73a6/api/src/main/java/org/openmrs/api/context/Daemon.java#L44). So its ok.